src/Avalonia.Controls/Notifications/WindowNotificationManager.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls.Primitives;
+using Avalonia.Interactivity;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Controls.Notifications
+{
+    public class WindowNotificationManager : TemplatedControl, IManagedNotificationManager
+    {
+        private IList _items;
+
+        public static NotificationPosition GetPosition(TopLevel obj)
+        {
+            return obj.GetValue(PositionProperty);
+        }
+
+        public static void SetPosition(TopLevel obj, NotificationPosition value)
+        {
+            obj.SetValue(PositionProperty, value);
+        }
+
+        public static readonly AvaloniaProperty<NotificationPosition> PositionProperty =
+          AvaloniaProperty.RegisterAttached<WindowNotificationManager, TopLevel, NotificationPosition>("Position", defaultValue: NotificationPosition.TopLeft, inherits: true);
```

Comment:
Why are `Position` and `MaxItems` attached properties? What controls other than `WindowNotificationManager` can they be attached to? Also why `inherits: true`?

Reply:
Its because you cant get hold of the WindowNotificationManager control in Xaml so you can do this:

```
<Window xmlns="https://github.com/avaloniaui" MinWidth="500" MinHeight="300"
        Title="Avalonia Control Gallery"
        Icon="/Assets/test_icon.ico"
        xmlns:local="clr-namespace:ControlCatalog"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="clr-namespace:ControlCatalog.ViewModels"
        xmlns:v="clr-namespace:ControlCatalog.Views"
        x:Class="ControlCatalog.MainWindow" WindowNotificationManager.MaxItems="3" WindowNotificationManager.Position="BottomRight">
```

Reply:
perhaps that not a good idea though? I kind of just added it as a conveniance.

Reply:
made them normal properties, this was a remnant from when user wasnt incharge of creating the WindowNotificationManager.